### DISCUSSION TITLE
在云服务器上的网站无法正常发送邮件

### DISCUSSION THREAD

**Discussion by bjx8970** [bs-community/blessing-skin-server#414] [Q&A]
![WF3AHPOZ{ 4ZP7ZK) SND4N](https://user-images.githubusercontent.com/107265702/173159740-b658280e-ee95-43a7-979c-cdb91f9951f4.png)
![O`HPX5XU 2OWB%5EO}X (ZH](https://user-images.githubusercontent.com/107265702/173159809-60348c74-77e3-4155-8779-4e79cd9265f4.png)
![3LNQM$UD_59C9R901%C1XU](https://user-images.githubusercontent.com/107265702/173159915-701cf4bf-adb9-464c-b5ae-46e434d26a65.png)

这样设置有没有问题？

**Comment by Qubik65536** [comment]
`MAIL_USERNAME` = `xxxxx.cpm`?

c***p***m?

**Reply by bjx8970** [reply]
哦哦，这个测试的打错了，不过正确情况下也发不出去，报错一样......

**Reply by ghost** [reply]
你的云主机是否已放通对应端口？

**Reply by bjx8970** [reply]
已经解决了，是对方云服务器默认锁定了所以邮件通讯端口，已经通过更换服务商解决，感谢回答

**Comment by bjx8970** [comment]
![OW$3AJ 5ZHYNHFNIG{996XT](https://user-images.githubusercontent.com/107265702/173160815-492ee03d-fbac-4db3-b90f-8bbcbea5bfa0.png)
